lms/djangoapps/course_news/templates/course_news/course-updates-fragment.html

```diff
@@ -0,0 +1,99 @@
+                    <h3 class="h3 week_holder__title">Программа курса</h3>
+                    % if blocks.get('children'):
+                        % for section in blocks.get('children'):
+                            <section class="week-structure">
+                                <div class="week-structure__name">
+                                    <span class="week-structure__ico"></span>
+                                    <strong>${ section['display_name'] }</strong>
+                                </div>
+                                <div class="week-structure__box">
+                                    <ul class="week-structure__list">
+                                    % for subsection in section.get('children', []):
+                                        <%
+                                        is_checked = ''
+                                        if subsection['last_accessed']:
+                                            is_checked = 'is-checked'
+                                        %>
+                                        <li class="week-structure__list-item">
+                                            <div class="week-structure__helper ${is_checked}"></div>
+                                            <a
+                                            class="outline-item focusable"
+                                            href="${ subsection['lms_web_url'] }"
+                                            id="${ subsection['id'] }"
+                                        >
+                                        ${ subsection['display_name'] }</a>
+                                        </li>
+                                    % endfor
+                                    </ul>
+                                </div>
+                            </section>
+                        % endfor
+                    % endif
+                </div>
+            </div>
+        </div>
+
+        % if plain_html_updates:
+            ${HTML(plain_html_updates)}
+        % else:
+            <div class="course-content">
+                <ul class="course-news">
+                    % if updates:
+                        % for index, update in enumerate(updates):
+                            <li class="course-news__item">
+                                <div class="course-news__title">
+                                    ${HTML(update.get("content_title"))}
+                                </div>
+                                % if not update.get("is_error"):
+                                    <div class="course-news__date">${update.get("date")}</div>
+                                % endif
+                                <div class="course-news__text">${HTML(update.get("content"))}</div>
+                                <div class="course-news__actions">
+                                    <button>
+                                        Подробнее
```

Comment:
The original term will always be in Russian, not being marked for i18n

If intended, then it's OK

Reply:
Last time I spoke to the client he asked if we could change the site to be displayed only in Russian, so it's not a problem I think